controllers/v1/reportsController.js

```diff
@@ -11,6 +11,11 @@ const FileStream = require(ROOT_PATH + '/generics/fileStream');
 const solutionsHelper = require(MODULES_BASE_PATH + '/solutions/helper');
 const reportsHelper = require(MODULES_BASE_PATH + '/reports/helper');
 let imageBaseUrl = '';
+const surveySubmissionsHelper = require(MODULES_BASE_PATH + '/surveySubmissions/helper');
+const questionsHelper = require(MODULES_BASE_PATH + '/questions/helper');
+const programsHelper = require(MODULES_BASE_PATH + '/programs/helper');
+const helperFunc = require('../../helper/chart_data');
+const solutionsQueries = require(DB_QUERY_BASE_PATH + '/solutions');
```

Comment:
@borkarsaish65 we need this? are we using it in controller? we should not have db query called in controllers

Reply:
removed now